.eslintignore

```diff
@@ -1 +1,3 @@
 lib
+commitlint.config.js
+fixture.js
```

Comment:
So this is an interesting issue. I discovered the `tsconfig.eslint.json` thing yesterday and forgot to update this PR because I was still running into issues with `fixture.js`. 

Turns out, if you have two files with the same name but just different `.ts/.js` endings, typescript will only pickup/include the `.ts` version of the file. It makes sense because typescript is about compiling to js, so it does not want naming conflicts. Having said that, how do you want to go about solving this issue? We have two files `fixture.ts` and `fixture.js` in the same folder. I am guessing we should rename one of them? Is there a reason we are keeping `fixture.js` around since it is all commented out code anyways?

Reply:
Those fixture files helped me when typescript-eslint was much more buggy. So I don't care about them much.

The picking up of only the `.ts` file... why would it occur when we have `outDir` specified?

In any case, feel free to remove any of them.

And I feel that it would be prudent to report this issue to typescript-eslint. Could you do that, please?

Reply:
Reported here: https://github.com/typescript-eslint/typescript-eslint/issues/890
I removed the `fixture.js` one since that seemed the least useful.